lib/license_finder/package_managers/pipenv.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'license_finder/package_utils/pypi'
+
+module LicenseFinder
+  class Pipenv < PackageManager
+    def initialize(options = {})
+      super
+      @lockfile = Pathname('Pipfile.lock')
+    end
+
+    def current_packages
+      dependencies['default'].map do |name, data|
```

Comment:
By default this is only parsing the `default` dependencies. I'm unsure if I should also parse out the `develop` dependencies as well. WDYT?